content/compose/reference/_index.md

```diff
@@ -126,12 +113,12 @@ relative to the current working directory.
 
 The `-f` flag is optional. If you don't provide this flag on the command line,
 Compose traverses the working directory and its parent directories looking for a
-`docker-compose.yml` and a `docker-compose.override.yml` file. You must supply
-at least the `docker-compose.yml` file. If both files are present on the same
+`compose.yml` and a `compose.override.yml` file. You must supply
+at least the `compose.yml` file. If both files are present on the same
```

Comment:
Both `docker-compose.yml` and `compose.yml` and corresponding override files are supported. Compose has some logic for deciding the lookup order of these files, which is what I think #18666 is asking for. How do these variants interact?

Again the bake file has a similar behavior so you can take a look at how I did it there: https://docs.docker.com/build/bake/reference/#file-format